src/language/formatter/formatter.ts

```diff
@@ -47,5 +47,5 @@ export interface IFormatterMetadata {
 export type ConsumerType = "human" | "machine";
 
 export interface IFormatter {
-    format(failures: RuleFailure[]): string;
+    format(failures: RuleFailure[], fixes?: RuleFailure[]): string;
```

Comment:
is the idea that `fixes` are only supplied to this method if `--fix` is enabled in the CLI? can you leave a comment explaining that?